cmd/monaco/dynatrace/dynatrace.go

```diff
@@ -45,7 +46,7 @@ func VerifyEnvironmentGeneration(envs manifest.Environments) bool {
 }
 
 func isClassicEnvironment(env manifest.EnvironmentDefinition) bool {
-	if _, err := version.GetDynatraceVersion(context.TODO(), rest.NewRestClient(auth.NewTokenAuthClient(env.Auth.Token.Value), rest.CreateRateLimitStrategy()), env.URL.Value); err != nil {
+	if _, err := version.GetDynatraceVersion(context.TODO(), rest.NewRestClient(auth.NewTokenAuthClient(env.Auth.Token.Value), nil, rest.CreateRateLimitStrategy()), env.URL.Value); err != nil {
```

Comment:
here as well, this means these initial API calls are missing from the traffic logs - smaller issue than the whole download but still